### PR TITLE
Add debug log for #2165

### PR DESCRIPTION
**Pull request by Waqar144**
Added a couple of debug statements to see if I can catch the issue.

### CODE REVIEW

File: src/helpers/codetohtmlconverter.cpp [hunk 1]
```diff
@@ -51,6 +53,7 @@ void CodeToHtmlConverter::initCodeLangs() Q_DECL_NOTHROW {
 }
 
 QString CodeToHtmlConverter::process(const QString &input) const {
+    qDebug() << "Going to highlight input: " << input.mid(0, 12) << ", with lang: " << _currentLang;
```
Comment (pbek) [inline src/helpers/codetohtmlconverter.cpp:56]: You don't need leading and trailing spaces, qDebug seems to add them on its own.

Comment (Waqar144) [reply src/helpers/codetohtmlconverter.cpp]: done

its only temporary though. Will remove these once the issue is identified.

Comment (pbek) [reply src/helpers/codetohtmlconverter.cpp]: Ah, OK. Because the whitespaces are also in above debug log. 😁

Comment (Waqar144) [reply src/helpers/codetohtmlconverter.cpp]: yeah, but does it really matter? (as said its temporary, just need it to confirm a few assumptions)

**Comment by Waqar144** [comment]
Lets hope this brings out something useful

**Pull request merged**